.pre-commit-config.yaml

```diff
@@ -35,17 +35,28 @@ repos:
     rev: "v0.5.6"
     hooks:
       - id: ruff
-        args: [--fix, --exit-non-zero-on-fix]
+        args: [
+          --fix,
+          --exit-non-zero-on-fix,
+          --ignore, E402,
+        ]
   - repo: https://github.com/PyCQA/flake8
     rev: 7.1.1
     hooks:
       - id: flake8
+        args: [
+          --max-line-length, "120",
+          --extend-ignore, E203 E402,
+        ]
+
   - repo: https://github.com/pycqa/pylint
     rev: v3.2.6
     hooks:
       - id: pylint
         args:
           - --output-format=colorized
+          - --disable=C0103,C0114,C0115,C0116,R0913,R1735,
+          - --max-line-length=120
```

Comment:
Same as above, not here! Otherwise running pylint outside pre-commit will show same errors (aka... vscode will stil complain!)